### PR TITLE
chore: prepare for v1.0.1 release

### PR DESCRIPTION
**Pull request by shreemaan-abhishek**
Prepare to release v1.0.1.

### CODE REVIEW

**Comment by wu-sheng** [comment]
Hi, this should be submitted by the release manager of SkyWalking.
You will not have svn and luarocks pushing auth.

**Comment by moonming** [comment]
> Hi, this should be submitted by the release manager of SkyWalking. You will not have svn and luarocks pushing auth.

I will do it after this PR merged

**Review by wu-sheng — APPROVED** [review]

**Pull request merged**
